domains/client/domain-executor/src/domain_block_processor.rs

```diff
@@ -501,6 +549,12 @@ where
         let mut bad_receipts_to_write = vec![];
 
         for execution_receipt in receipts.iter() {
+            // TODO: check genesis receipt and fix https://github.com/subspace/subspace/issues/1301 to
+            // produce fraud proof accordingly
```

Comment:
This TODO is invalid as the genesis receipt will be a truth on the consensus chain, generated on the domain instantiation, there is no fraud proof for the genesis receipt, which invalidates #1301 naturally.